terminusdb_client/woqlclient/woqlClient.py

```diff
@@ -1120,15 +1158,21 @@ def _prepare_revision_control_args(self, rc_args=None):
         Returns
         -------
         dict or bool
-            ``False`` if ``rc_args`` is not provided, other the modified ``rc_args``.
+            ``False`` if ``rc_args`` is not provided, otherwise the modified ``rc_args``.
         """
         if rc_args is None:
             return False
         if not rc_args.get("author"):
             rc_args["author"] = self.conCapabilities.author()
         return rc_args
 
-    def dispatch(self, action, url, payload=None, file_dict=None):
+    def dispatch(
+        self,
+        action: Union[str, Tuple[str]],
```

Comment:
Shouldn't `APIEndpointConst` be an `enum`? I think it would make sense to help stricter type checking.